docs/middleware.md

```diff
@@ -184,6 +183,31 @@ The following arguments are supported:
 
 The middleware won't GZip responses that already have a `Content-Encoding` set, to prevent them from being encoded twice.
 
+## LimitRequestMiddleware
```

Comment:
I’d put Size in the name

Reply:
Yeah, my thoughts were to make it possible to add more stuff on this middleware, but I've added the "Size".

Reply:
Ah no. It can't be `RequestSize`... Either `LimitRequestBodySizeMiddleware`, or `LimitRequestMiddleware`. Or anything that says "payload" or "body".

Reply:
I like `LimitRequestBodySizeMiddleware`